recipe-server/client/control_new/components/common/ShieldIdenticon.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ShieldIdenticon extends React.PureComponent {
+  static propTypes = {
+    seed: PropTypes.string.isRequired,
+    size: PropTypes.number,
+    className: PropTypes.string,
+  }
+
+  static defaultProps = {
+    size: 64,
+    className: null,
+  }
+
+  render() {
+    const { seed, size, className } = this.props;
+
+    return (
+      <img
+        className={className}
```

Comment:
Instead of explicitly using the `className` prop, we can just do something like:

```
const { seed, size, ...rest } = this.props;
return (<img src="..." height={size} width={size} {...rest} />)
```

This pulls out `seed` and `size` from props and then groups any other props as `rest`. We pass this down the line so we don't need to worry about things like `className`, `onClick`, etc.

Reply:
I advised @marcrowo not to do this, since we've had instances in the past of too many props being passed down to HTML components, causing errors. This was mostly with regards to redux-form. I think having a whitelist of props to pass is a better policy when dealing with HTML components, which are strict about which props they accept.